### ISSUE TITLE
Accessibility for the reference line

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/photoelectric-effect#234]
There is a TODO pointing to this component, from https://github.com/phetsims/photoelectric-effect/issues/228.

```
    // TODO: Consider accessibility for this. There is some discussion about whether this should be a "slider"
    //   or a custom draggable. What should it be?
```

Right now it is a custom draggable. I think we decided to keep that as is. If so, is the custom keyboard input working as desired?

**Comment by brettfiedler** [comment]
Can we just check to see if we're doing the same implementation as Quantum Bound States?

**Comment by brettfiedler** [comment]
We don't need the "Jump to Zero" shortcut from QBS.

**Comment by jessegreenberg** [comment]
This was merged into main, closing.
